serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/service/elma/ELMALookupService.java

```diff
@@ -41,4 +43,32 @@ public Endpoint lookup(String organisationNumber) throws EndpointUrlNotFound {
             throw new EndpointUrlNotFound(String.format("Failed lookup through ELMA of %s", organisationNumber), e);
         }
     }
+
+    public boolean identifierHasInnsynskravCapability(String identifier) {
+        try {
+            Endpoint ep = lookupClient.getEndpoint(ParticipantIdentifier.of(identifier),
+                    DocumentTypeIdentifier.of(props.getElmaDPEInnsyn().getDocumentTypeIdentifier()),
+                    ProcessIdentifier.of(props.getElmaDPEInnsyn().getProcessIdentifier()),
+                    transportProfile);
+            return !isNullOrEmpty(ep.getAddress().toString());
+        } catch (PeppolSecurityException e) {
+            throw new ServiceRegistryException(e);
+        } catch (LookupException | EndpointNotFoundException e) {
+            return false;
+        }
+    }
+
+    public boolean identifierHasInnsynDataCapability(String organisationNumber) {
+        try {
+            Endpoint ep = lookupClient.getEndpoint(ParticipantIdentifier.of(organisationNumber),
+                    DocumentTypeIdentifier.of(props.getElmaDPEData().getDocumentTypeIdentifier()),
```

Comment:
Burde kanskje trekke ut denne koden i en metode for å unngå kodeduplisering med identifierHasInnsynskravCapability